src/PGURE-SVT.cpp

```diff
@@ -156,7 +156,7 @@ int main(int argc, char** argv) {
 	}
 	
 	// Block overlap
-	int Bo = 1;
+	int Bo = 4;
```

Comment:
This works in terms of memory, speed etc. but it is taking the wrong patches in the motion estimation step. Need to work out how to fix this.